.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "submodules/FileSystem"]
+	path = submodules/FileSystem
+	url = git@github.com:aspnet/FileSystem.git
```

Comment:
Seems there is no way to add a fallback. We'll leave it for now and see whether it can lead to some inconvenience.